Module 10 - Javascript and Ajax/pages/DeleteStudentWithRemoting.page

```diff
@@ -0,0 +1,42 @@
+<apex:page standardController="Student__c" recordSetVar="studentDetails" extensions="DeleteStudentWithRemotingController" showHeader="false" >
+    <script>
+    function remoteDelete() 
+    {
+        Visualforce.remoting.Manager.invokeAction(
+            									'{!$RemoteAction.DeleteStudentWithRemotingController.deleteStudents}',
+            									'{! listofStudents}',
+            									handleResult
+        										 );
+    }
+  
+    function handleResult(result, event)
+    {
+    	if(event.status)
```

Comment:
how can you judge is the status is present it was success?
status can be 404, 500